utils/bloom_filter.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/base64"
+
+	"github.com/bits-and-blooms/bloom"
+	"github.com/forta-network/forta-core-go/protocol"
+)
+
+const (
+	NumMaxAddresses = 50
```

Comment:
I see that we expose this constant to use in other places. I see a few points worth revisiting:
- Can we rename this to `NumMaxAlertBloomAddresses` or similar? When importing it like `utils.NumMaxAddresses`, it doesn't make much sense why we should be referring to this constant from elsewhere.
- I see that we use the same number in node but we are not reusing this constant. Should we do that?
- Another option could be not exposing this constant, containing the truncation logic here in an exposed method like `TruncateAlertBloomAddresses()` and reuse in node and analyzer. Then the code is simpler and common in both places. Do you think this is a good idea?

Reply:
- `NumMaxAlertBloomAddresses` would be misleading because it's not limiting the number of addresses in the filter but rather the alert. But i agree that it could be improved. Maybe `NumMaxAddressesPerAlert`?
- yeap
- IDK, maybe? I wouldn't mess with the analyzer's logic because it currently stops appending instead of truncating. If we were to use a truncation logic after all is done, we'd need to match addresses and address details. Which I think would be tedious and add unnecessary complexity. We can move it to the core and use that in the node, but not sure if it worths the trouble